synth-publish.go

```diff
@@ -220,8 +221,41 @@ func checkPublishingStatus(latest postedData, result chan<- publicationResult, s
 
 	switch resp.StatusCode {
```

Comment:
Just a recommendation, since in the script there're already functions i'd use a func for each case:

Reply:
There is "latest.tid" here which is only valid in checkPublishingStatus function so I've decided to not use such function